dev-libs/Ice/metadata.xml

```diff
@@ -1,13 +1,10 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<maintainer type="person">
 		<email>Dessa@gmake.de</email>
 		<name>Robert Förster</name>
 	</maintainer>
-	<maintainer type="project">
-		<email>cpp@gentoo.org</email>
```

Comment:
proxy-maint?

Reply:
Are you asking me? I don't know how you were committing.

Reply:
my understanding was that it is not allowed to be a external maintainer when not under someones umbrella. i could be wrong on that tho (or maybe i should just work on my quizzes :P )

@SoapGentoo did review my initial bump but since then it got a non-maintainer bump anyway so im not 100 % sure